### PR TITLE
Adobe Target v2 Extension v0.19.2 release notes

### PR DESCRIPTION
**Pull request by jasonwaters**
updated release notes for Adobe Target v2 Extension v0.19.2

### CODE REVIEW

File: help/tags/extensions/client/target-v2/release-notes.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ exl-id: c1a04e62-026d-4b16-aa70-bc6d5dbe6b2d
 >
 >Adobe Experience Platform Launch has been rebranded as a suite of data collection technologies in Adobe Experience Platform. Several terminology changes have rolled out across the product documentation as a result. Please refer to the following [document](../../../term-updates.md) for a consolidated reference of the terminology changes.
 
+## v0.19.2 (February 14, 2023)
+
+- Bug fix: allow timeout to be set to a data element
```
Comment (hbhandal80) [inline help/tags/extensions/client/target-v2/release-notes.md]: Can we update this to: 

"Fixed an issue with allowing timeout to be set to a data element."

**Review by hbhandal80 — CHANGES_REQUESTED** [review]
A comment left.

File: help/tags/extensions/client/target-v2/release-notes.md (hunk 1 above)
Comment (jasonwaters) [reply help/tags/extensions/client/target-v2/release-notes.md]: Thank you for the feedback @hbhandal80 !  I have made the change.

**Review by hbhandal80 — APPROVED** [review]
LGTM :)

**Comment by vgiurgiu** [comment]
Hi @jasonwaters and @hbhandal80 - just a suggestion - can we please use the corporate GitHub repository for any documentation updates submitted by internal Adobe employees? That would simplify and expedite the documentation update process. Thanks!

**Pull request merged**